plugins/by-name/dap-go/default.nix

```diff
@@ -0,0 +1,52 @@
+{
+  lib,
+  ...
+}:
+let
+  inherit (lib) types;
+  inherit (lib.nixvim) defaultNullOpts;
+
+  dapHelpers = import ../dap/dapHelpers.nix { inherit lib; };
+in
+lib.nixvim.plugins.mkNeovimPlugin {
+  name = "dap-go";
+  package = "nvim-dap-go";
+
+  maintainers = [ lib.maintainers.khaneliman ];
+
+  settingsOptions = {
+    dap_configurations = lib.nixvim.mkNullOrOption (types.listOf dapHelpers.configurationOption) ''
+      Additional dap configurations.
+      See `:h dap-configuration` for more detail.
+    '';
+
+    delve = {
+      path = defaultNullOpts.mkStr "dlv" "The path to the executable dlv which will be used for debugging.";
+
+      initialize_timeout_sec = defaultNullOpts.mkInt 20 "Time to wait for delve to initialize the debug session.";
+
+      port = defaultNullOpts.mkStr "$\{port}" ''
```

Comment:
Is this `"$\{port}"` correct?

I don't think that's how you escape a nix interpolation, which feels like it may have been the original intent.

Same for the description below, although we probably shouldn't be documenting the default within the description anyway.

Reply:
I think the description explaining the default just adds detail explaining what it means to have that default.